lighthouse-viewer/app/styles/viewer.css

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --lh-background-color: rgb(32,50,70);
+}
+
+.drop_zone {
+  display: flex;
```

Comment:
IMO we shouldn't sort until we have tooling to enforce and reformat. And I don't think its a huge priority yet, since our UI surface area is pretty contained.